offload/cmake/caches/AMDGPUBot.cmake

```diff
@@ -5,6 +5,7 @@ set(CMAKE_INSTALL_PREFIX /tmp/llvm.install.test CACHE STRING "")
 
 # General settings
 set(CMAKE_BUILD_TYPE Release CACHE STRING "")
+set(BUILD_SHARED_LIBS ON CACHE BOOL "")
```

Comment:
What is the compile + test time difference with this?

Reply:
neglectable on the buildbots